homeassistant/components/binary_sensor/knx.py

```diff
@@ -49,6 +50,7 @@
     vol.Optional(CONF_DEVICE_CLASS): cv.string,
     vol.Optional(CONF_SIGNIFICANT_BIT, default=CONF_DEFAULT_SIGNIFICANT_BIT):
         cv.positive_int,
+    vol.Optional(CONF_RESET_AFTER, default=None): cv.positive_int,
```

Comment:
Remove default `None`. Default value must pass validation in latest voluptuous version.

Reply:
will change it to:

```
+    vol.Optional(CONF_RESET_AFTER, default=None): vol.Any(
+        None, cv.positive_int),
```